packages/bot-web-ui/src/assets/sass/load-modal.scss

```diff
@@ -392,7 +392,7 @@ $modal_container_height: calc(100vh - 210px);
     position: relative;
     border-radius: $BORDER_RADIUS;
     border: solid 1px var(--border-normal);
-    height: 100%;
+    height: 95%;
```

Comment:
Why `95%`?

Reply:
I wanted to make the height shorter and I thought 95 is enough. Although i could not see the result in safari.